manager/controllers/cves/images.go

```diff
@@ -0,0 +1,123 @@
+package cves
+
+import (
+	"app/base/utils"
+	"app/manager/base"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"gorm.io/gorm"
+)
+
+var getCveImagesAllowedFilters = []string{
+	base.SearchQuery,
+	base.DataFormatQuery,
+}
+
+var getCveImagesFilterArgs = map[string]interface{}{
+	base.SortFilterArgs: base.SortArgs{
+		SortableColumns: map[string]string{
+			"id":       "repository.id",
+			"name":     "repository.repository",
+			"registry": "repository.registry",
+		},
+		DefaultSortable: []base.SortItem{{Column: "id", Desc: false}},
+	},
+	base.SearchQuery: base.ImagesSearch,
+}
+
+// GetCveImagesSelect
+// @Description Exposed images in cve data
+// @Description presents in response
+type GetCveImagesSelect struct {
+	ExposedClusters *int32              `json:"exposedClusters" csv:"exposedClusters" gorm:"column:exposed_clusters"`
+	Repository      *string             `json:"name" csv:"name" gorm:"column:repository"`
+	Registry        *string             `json:"registry" csv:"registry" gorm:"column:registry"`
+	Version         *utils.ImageVersion `json:"version" csv:"version" gorm:"column:tags"`
+}
+
+type GetCveImagesResponse struct {
+	Data []GetCveImagesSelect `json:"data"`
+	Meta interface{}          `json:"meta"`
+}
+
+// GetCveImages represents /cves/{cve_name}/exposed_images endpoint controller.
+//
+// @id GetCveImages
+// @summary List of images affecting single cve
+// @security RhIdentity || BasicAuth
+// @Tags cves
+// @description Endpoint returning images affecting the given single cluster
+// @accept */*
+// @produce json
+// @Param cve_name        path  string   true  "cve ID"
+// @Param sort            query []string false "column for sort"                                      collectionFormat(multi) collectionFormat(csv)
+// @Param search          query string   false "image name/registry search"                           example(ubi8)
+// @Param limit           query int      false "limit per page"                                       example(10) minimum(0) maximum(100)
+// @Param offset          query int      false "page offset"                                          example(10) minimum(0)
+// @Param data_format     query string   false "data section format"                                  enums(json,csv)
+// @Param report          query bool     false "overrides limit and offset to return everything"
+// @router /cves/{cve_name}/exposed_images [get]
+// @success 200 {object} GetCveImagesResponse
+// @failure 400 {object} base.Error
+// @failure 404 {object} base.Error "cve does not exist"
+// @failure 500 {object} base.Error
+func (c *Controller) GetCveImages(ctx *gin.Context) {
+	accountID := ctx.GetInt64("account_id")
+	cveName := ctx.Param("cve_name")
+
+	if cveName == "" {
```

Comment:
There could be check if cve exists instead - like in https://github.com/RedHatInsights/vuln4shift-backend/blob/master/manager/controllers/cves/exposed.go#L88-L106, to be consistent